corehq/pillows/blobs.py

```diff
@@ -0,0 +1,26 @@
+from couchexport.models import SavedBasicExport
+from pillowtop.listener import BasicPillow
+
+
+class BlobDeletionPillow(BasicPillow):
+
+    include_docs = False
+    doc_types = {cls.__name__: cls for cls in [
+        # list of all model classes that use BlobMixin
+        SavedBasicExport,
+    ]}
+
+    def change_trigger(self, changes_dict):
+        if not changes_dict.get('deleted', False):
+            return None
+        doc = self.db.get(changes_dict['id'])
```

Comment:
Went to voice with @czue and decided to change the structure of the blob db so we can find/delete attachments for a document using only the doc id and database name. Adding "don't merge" label until I have that implemented.